git/util.py

```diff
@@ -52,7 +52,18 @@
            'RemoteProgress', 'CallableRemoteProgress', 'rmtree', 'unbare_repo',
            'HIDE_WINDOWS_KNOWN_ERRORS')
 
+
+try:  # Python 2.7+
+    import logging.NullHandler
+except ImportError:
+    class NullHandler(logging.Handler):
```

Comment:
I juts following the same pattern of enable logging like it is done in other part of `GitPython` as well (see `git/cmd.py` for example). The only difference is that unit tests under Python 2.6+ are failing because the `logging` package in 2.6 doesn't have the `NullHandler` thus this little piece of code.